joomla/joomla-cms
# [installation] Update also modified by user id field (and also new fields table)

### Summary of Changes

This PR (a continuation of https://github.com/joomla/joomla-cms/pull/11245) makes installation also update the `modified_by` user id field with the super user id in all tables of joomla + sample data.

### Testing Instructions

1. Apply patch
2. Delete configuration.php
3. Install Joomla "sample blog"
4. After install check there is no `JUser: :_load: Unable to load user with ID: 713` when editing content.

### Documentation Changes Required

None.
